pkg/proxy/handlers/spacelister.go

```diff
@@ -36,9 +38,10 @@ func NewSpaceLister(app application.Application) *SpaceLister {
 }
 
 func (s *SpaceLister) HandleSpaceListRequest(ctx echo.Context) error {
-
+	startTime := ctx.Get(context.StartTime).(time.Time)
 	workspaces, err := s.ListUserWorkspaces(ctx)
 	if err != nil {
+		metrics.RegServProxyResponseHistogramVec.WithLabelValues(metrics.ResponseMetricLabelReject).Observe(time.Since(startTime).Seconds())
```

Comment:
is there any reason for using seconds as the unit? wouldn't miliseconds be better?

Reply:
the default unit for buckets, and thus observe is seconds